<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>292</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ElectroLogo1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Login_lineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_lineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Â© Designed by Philipp Lazarev, 2020</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resourses</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElectro_Inc">
    <property name="title">
     <string>Electro Inc.</string>
    </property>
   </widget>
   <addaction name="menuElectro_Inc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
